truffle/src/main/ruby/core/rubinius/common/main.rb

```diff
@@ -34,6 +35,22 @@ def include(*mods)
   # Truffle: Do not define #public, #private in Ruby since they need
   # to set the top-level frame visibility when called with no argument.
```

Comment:
Aren't we ok now we don't overwrite Java methods with Ruby ones? So are the comments below still required?